test/verify/test_gemm_add.cpp

```diff
@@ -57,4 +57,3 @@ struct test_gemm_add : verify_program<test_gemm_add<DType>>
 
 template struct test_gemm_add<migraphx::shape::float_type>;
 template struct test_gemm_add<migraphx::shape::half_type>;
-template struct test_gemm_add<migraphx::shape::fp8e4m3fnuz_type>;
```

Comment:
instead of removing this line, comment out and add `TODO above with issue  number`

Reply:
Done.